iac-extensions/kubernetes/src/test/resources/checks/ClearTextProtocolsCheck/clearTextProtocolsTestTemplate.yaml

```diff
@@ -0,0 +1,38 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: curl
+spec:
+  template:
+    spec:
+      containers:
+        - name: curl
+          image: curlimages/curl
+          command: ["curl"]
+          args: ["http://example.com/"] # Noncompliant {{Make sure that using clear-text protocols is safe here.}}
```

Comment:
Please include tests that show case insensitivity and also tests where the URL is in another scalar other than args.
Example:
```
  containers:
    - name: applications
      image: my_image_with_ssh
      ports:
        - hostIP: "ftp://example.com/"
```
Doesn't make the most sense in the file itself (maybe you find a better example directly from the already raised its) but it shows, that these can be found at other places and not only in lists like `[" "]`.

Reply:
Also this test could also be nice: `args: ["-v", "http://example.com/"]`